dNet/AuthPackets.cpp

```diff
@@ -229,7 +229,7 @@ void AuthPackets::SendLoginResponse(dServer* server, const SystemAddress& sysAdd
 	RakNet::BitStream loginResponse;
 	BitStreamUtils::WriteHeader(loginResponse, eConnectionType::CLIENT, eClientMessageType::LOGIN_RESPONSE);
 
-	loginResponse.Write<uint8_t>(GeneralUtils::CastUnderlyingType(responseCode));
```

Comment:
Is there a particular reason this is cast in the first place?  the enum should be the correct base type already no?

Reply:
You're right. Looks like the call is redundant here.